pkg/server/admin.go

```diff
@@ -554,7 +554,21 @@ func (s *adminServer) TableStats(
 
 	// Construct TableStatsResponse by sending an RPC to every node involved.
 	tableStatResponse := serverpb.TableStatsResponse{
-		NodeCount:  int64(len(nodeIDs)),
+		NodeCount: int64(len(nodeIDs)),
+		// TODO(mrtracy): The "RangeCount" returned by TableStats is more
```

Comment:
The action is "We should consider plumbing TableStats' meta2 query through the DistSender so that it will share the advantage of populating the cache (without the disadvantage of potentially returning stale data)."  Considering it is the action - i'm not sure it actually is a better thing to do.

This was just a thought, from someone else, about a minor potential performance tweak to a non-critical system. At this point, it's clear that making it a "TODO" just makes it into a bigger deal than it is, and I'm considering just deleting the TODO altogether.